docker/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+services:
+  nymead:
+    build: .
+    image: nymead-image
+    container_name: nymead-container
+    network_mode: host
+    privileged: true
+    hostname: nymea-docker
+    ports:
+      - "80:80"
+      - "443:443"
+      - "2223:2223"
+      - "2224:2224" #tcp
```

Comment:
would 2223 be nymea:// and 2224 be nymeas:// ?

isn't 2222 and 2223 the default?